Rakefile

```diff
@@ -1,7 +1,7 @@
 require "logstash/devutils/rake"
 
 desc "Get jars"
-task :get_jars do
+task :vendor do
```

Comment:
@jordansissel does this all make sense the way its done here? Is there a specific refactor you'd like to see?